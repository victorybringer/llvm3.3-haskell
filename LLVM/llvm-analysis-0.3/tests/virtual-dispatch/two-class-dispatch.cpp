struct DispatchBase {
  DispatchBase();
  virtual void f();
};

struct Derived : public DispatchBase {
  Derived();
  virtual void f();
};

Derived::Derived(): DispatchBase() {
}

DispatchBase::DispatchBase() {
}

void DispatchBase::f() {

}

void Derived::f() {

}

void test(DispatchBase * b) {
  b->f();
}
