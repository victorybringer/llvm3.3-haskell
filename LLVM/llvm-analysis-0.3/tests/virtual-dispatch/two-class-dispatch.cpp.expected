fromList [ ("test", fromList ["DispatchBase::f", "Derived::f"])]
